go/ct/gen/state_test.go

```diff
@@ -342,6 +342,7 @@ func TestStateGenerator_ClonesAreIndependent(t *testing.T) {
 		"stack={2≤size≤200}",
 		"memory={}",
 		"storage={}",
+		"transient={}",
```

Comment:
in at least one of the states, add a condition, so that we can check it is properly printed.